packages/salesforcedx-sobjects-faux-generator/src/generator/soqlMetadataGenerator.ts

```diff
@@ -15,8 +15,8 @@ const outputFolderPath = path.join(projectPaths.toolsFolder(), SOQLMETADATA_DIR)
 export const generateAllMetadata = async (sobjects: SObjectsStandardAndCustom) => {
   await createDirectory(outputFolderPath);
 
-  await Promise.all([
-    ...Object.entries(sobjects)
+  await Promise.all(
+    Object.entries(sobjects)
```

Comment:
prevents unnecessary spread (helps you know when a function takes an interable without needing to be spread first)